src/apis/apis.js

```diff
@@ -0,0 +1,16 @@
+import axios from "../axios";
+// such a function is reusable, more like this can be made
+export const handleAppMetricsPostRequest = (data,endpoint) => {
```

Comment:
@khalifan-kfan this is awesome, hopefully, in the subsquent sprints we can refactor it to make it more reusable beyond just the appMetricsPost but have a generic post function or rather one function that could take in 3 variables, the URL, the HTTP method, and if any the object data. This is good work.

Reply:
Yes @neelxie, so this file may also have like 3 types of generic post functions as well depending on the end-points requirement